README.md

```diff
@@ -23,45 +34,60 @@ To run Unvanquished Tray in the foreground, execute:
 unvtray
 ```
 
-or, if the above does not work (e.g. when `~/.local/bin` is not in your `PATH`):
+or, if the above does not work (e.g. when `~/.local/bin` is not in your `PATH`
+on Unix-like platforms):
 
 ```
 python -m unvtray
 ```
 
-To start it with your window manager, append ` &` as necessary (for running in
-the background) and add the command to your startup file (e.g. `.xinitrc`).
+### Autostart
 
-## Troubleshooting
+#### on GNU/Linux or BSD
 
-### Left click on the icon does not launch the game
+To either of the above commands, append `&` as necessary (for running in the
+background) and add it to your display server startup file (e.g. `~/.xinitrc`
+if starting Xorg manually or `~/.xprofile` if using a display manager), or to
+your desktop environment or window manager configuration (e.g. `.i3/config` for
+i3). See the [Arch Linux
+wiki](https://wiki.archlinux.org/title/autostarting#On_Xorg_startup) for more
+options.
 
-On GNU/Linux, this is a limitation of the `pystray` library's default backend;
-use the context menu (right click) to connect.
+## Troubleshooting
 
-### The game does not launch even when selecting a server from the context menu
+### The game does not launch when selecting a server from the context menu
 
-#### GNU/Linux
+#### on GNU/Linux or BSD
 
 Make sure `xdg-utils` are installed and that your distribution of Unvanquished
-comes with a `.desktop` file that describes the `unv://` URI scheme.
+comes with a `.desktop` file that describes the `unv://` URI scheme. For
+example, both the [official
+updater](https://github.com/Unvanquished/updater/releases/) (all platforms) and
+the [AUR package](https://aur.archlinux.org/packages/unvanquished) (Arch Linux)
```

Comment:
One can also mention our official [flatpak](https://flathub.org/apps/details/net.unvanquished.Unvanquished) that also properly configure the `unv://` handler.

Reply:
Is it correct to say that the flatpak "installs a `.desktop` file"?

Reply:
Yes, this is exactly the way flatpak sets up the `unv://` handler:

https://github.com/flathub/net.unvanquished.Unvanquished/blob/d81f8ee70d975fe56ae55b84639439775a17cf66/net.unvanquished.Unvanquished.desktop#L8